httpx/decoders.py

```diff
@@ -85,6 +85,9 @@ class BrotliDecoder(Decoder):
     Handle 'brotli' decoding.
 
     Requires `pip install brotlipy`. See: https://brotlipy.readthedocs.io/
+        or   `pip install brotli`. See https://github.com/google/brotli
+    Supports both 'brotlipy' and 'Brotli' packages since they share an import
+    name. The top branches are for 'brotlipy' and bottom branches for 'Brotli'
```

Comment:
GitHub won't let me add the comment in the right place, but in line 96 below there's an `assert` that you may want to amend.

Reply:
Good catch, thanks !